src/Servers/Kestrel/Transport.MsQuic/src/Internal/UIntExtensions.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal
+{
+    internal static class UIntExtensions
+    {
+        internal static bool Succeeded(this uint status)
```

Comment:
Don't put extensions methods on primitive types, it gets really confusing when that shows up in intellisense for the wrong field. Make it a normal helper method.